build.sbt

```diff
@@ -6,25 +6,25 @@ val appName = "http-caching-client"
 
 val compileDependencies = Seq(
   "com.typesafe.play" %% "play-json"       % "2.5.15" % "provided",
-  "uk.gov.hmrc"       %% "json-encryption" % "3.2.0",
+  "uk.gov.hmrc"       %% "json-encryption" % "4.0.0",
   "uk.gov.hmrc"       %% "http-core"       % "0.5.0"
```

Comment:
We can get rid of http-core and use http-verbs instead

Reply:
Done.